frontend/src/components/doctors-filtration-panel/doctors-filtration-panel.tsx

```diff
@@ -0,0 +1,159 @@
+import * as React from 'react';
+import { useForm } from 'react-hook-form';
+import styles from './doctors-filtration.module.scss';
+import location from 'assets/images/location.svg';
+import specialty from 'assets/images/specialty.svg';
+import clinic from 'assets/images/clinic.svg';
+import { TextInput, Checkbox } from 'components/common';
+import { DoctorType, ClinicType, InputType, InputColor, DoctorFiltration } from 'common/enums';
+import { IDoctorFiltrationPayload, IDoctorCheckbox } from 'common/interfaces';
+import Details from 'components/common/details/details';
+
+const DEFAULT_VALUES: IDoctorFiltrationPayload = {
+  [DoctorFiltration.SEARCH]: '',
+  [DoctorFiltration.CITY]: '',
+  [DoctorFiltration.SPECIALTY]: DoctorType.SURGEON,
+  [DoctorFiltration.TYPE_OF_CLINIC]: ClinicType.STATE
+}
+
+const CHECKBOXES_DEFAULT_VALUES: IDoctorCheckbox = {
+  [DoctorType.PEDIATRICIAN]: false,
+  [DoctorType.ENDOCRINOLOGIST]: false,
+  [DoctorType.DENTIST]: false,
+  [DoctorType.SURGEON]: false,
+  [DoctorType.DERMATOLOGIST]: false,
+  [ClinicType.PRIVATE]: false,
+  [ClinicType.STATE]: false
+}
+
+const DoctorsFiltration: React.FC = () => {
+  const [checkboxesValues, setCheckboxesValues] = React.useState(CHECKBOXES_DEFAULT_VALUES);
+
+  const { handleSubmit, control, errors } = useForm<IDoctorFiltrationPayload>({
+    defaultValues: DEFAULT_VALUES,
+    mode: "onChange"
+  });
+
+  const onSubmit = (data: IDoctorFiltrationPayload) => {
+    console.log(data);
+  };
+
+  const onCheckboxChange = (name: keyof IDoctorCheckbox) => {
+    setCheckboxesValues({
+      ...checkboxesValues,
+      [name]: !checkboxesValues[name]
+    })
+  }
```

Comment:
```suggestion
  const onSubmit = (data: IDoctorFiltrationPayload) => {
    console.log(data);
  };

  const onCheckboxChange = (name: keyof IDoctorCheckbox) => {
    setCheckboxesValues({
      ...checkboxesValues,
      [name]: !checkboxesValues[name]
    })
  }
```
use handle<Element><Event> for local component cb's